kratos/sources/model_part_io.cpp

```diff
@@ -3637,6 +3639,29 @@ void  ModelPartIO::ReadSubModelPartConditionsBlock(ModelPart& rMainModelPart, Mo
     KRATOS_CATCH("")
 }
 
+void  ModelPartIO::ReadSubModelPartGeometriesBlock(ModelPart& rMainModelPart, ModelPart& rSubModelPart)
+{
+    KRATOS_TRY
+
+    SizeType geometry_id;
+    std::string word;
+    std::vector<SizeType> ordered_ids;
+
+    while (!mpStream->eof())
+    {
+        ReadWord(word); // Reading the node id or End
```

Comment:
Wrong comment

Reply:
Copy & paste error. Thanks for spotting.